hooks/README.md

```diff
@@ -1,3 +1,25 @@
 # OpenFeature Java Hooks
 
-Hooks are a mechanism whereby application developers can add arbitrary behavior to flag evaluation. They operate similarly to middleware in many web frameworks. Please see the [spec](https://github.com/open-feature/spec/blob/main/specification/sections/04-hooks.md) for more details.
+The OpenTelemetry hook for OpenFeature provides
+a [spec compliant] (https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/feature-flags.md)
+way to automatically add a feature flag
+evaluation to a span as a span event. This can be used to determine the impact a feature has on a request,
+enabling enhanced observability use cases, such as A/B testing or progressive feature releases.
+
+## Installation
+
+```
+    <groupId>dev.openfeature.contrib.hooks</groupId>
+	<artifactId>otel</artifactId>
+	<version>0.4.0</version> 
```

Comment:
This version should be controlled with release please. @toddbaert should be able to provide some examples.

Reply:
Yep! Just add annotations as seen [here](https://github.com/open-feature/java-sdk-contrib/blob/8fe9ebd3f0491098e22ac195d7d9e2347fec1cc6/providers/flagd/README.md?plain=1#L8-L16) - it will make sure the version is updated in your example as we release.

Reply:
The Release Please config will also need to be updated to include the readme.

https://github.com/open-feature/java-sdk-contrib/blob/main/release-please-config.json#L31